src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrPatchHandler.cs

```diff
@@ -197,19 +228,16 @@ public class VmrPatchHandler : IVmrPatchHandler
                 patchName,
                 $"{sha1}..{sha2}",
                 "--",
-                ".", // Apply for a given directory (we will call this from the content dir)
+                path, // Apply for a given file or directory (we will call this from the content dir)
```

Comment:
Can we update the XML doc on the additional mappings member of the Source-Mappings too?

Reply:
Later we can bring this string over to the json file too

Reply:
I'll add a explanation that it can be a file or a folder

Reply:
And that they have to match